mage_ai/orchestration/db/models/schedules.py

```diff
@@ -313,7 +314,7 @@ def in_progress_runs(
                 self.PipelineRunStatus.INITIAL,
                 self.PipelineRunStatus.RUNNING,
             ]),
-            PipelineRun.passed_sla.is_(False),
+            (coalesce(PipelineRun.passed_sla, False) == False),  # noqa: E712
```

Comment:
just curious, what's the reason for doing this?

Reply:
SQLAcademy will translate the original filter condition to the following illegal SQL statement syntax as MSSQL does not have a native built-in boolean field: it has a BIT (tiny integer) data type for boolean.
```
passed_sla is 0
```
The new filter statement would be translated into a valid SQL statement in MSSQL.
```
COALESCE(passed_sla, 0) == 0
```
This expression also added a fix based on the pipeline running logic: when a pipeline run starts initially, `passed_sla` has a `null` value, which should be treated as 0 (False).